ansible/vagrant/vagrant-ansible.yml

```diff
@@ -1,19 +1,20 @@
 - hosts: all
   sudo: yes
+  vars:
+    - public_iface: eth0
```

Comment:
@eparis great suggestion! :) In this way we can also be consistent with the legacy default. Nice!